### DISCUSSION TITLE
Best way to update

### DISCUSSION THREAD

**Discussion by ramair02** [mza921/Plex-Auto-Collections#223] [Q&A]
This is a dumb question, but asking for someone not super comfortable with terminal -- me :)

What's the best way to update Plex-Auto-Collections to the latest version? In the past, I've just downloaded the latest and extracted to the folder where it lives. Then I run `pip install -r /path/to/requirements.txt`

I recently moved from Windows to Unraid and am now using the User Scripts app to run the script on a schedule. I'm curious if there is a script I can run when new versions are released using User Scripts.

**Comment by tehniemer** [comment]
I would recommend to start learning Docker now that you are on Unraid. It's a bit of a steep learning curve, but [this](https://github.com/htpcBeginner/docker-traefik) repo and community is a very good place to start.

**Reply by meisnate12** [reply]
I have a [unRAID installation guide](https://github.com/meisnate12/Plex-Meta-Manager/wiki/unRAID-Installation) for a program I wrote that might help you out.

**Reply by ramair02** [reply]
> I would recommend to start learning Docker now that you are on Unraid. It's a bit of a steep learning curve, but [this](https://github.com/htpcBeginner/docker-traefik) repo and community is a very good place to start.

I am using a bunch of different Dockers on unraid. But since this is simply a script, I thought the best approach was user scripts. It works well enough. What am I missing?

**Reply by tehniemer** [reply]
Just the ease of updating mainly, set it and forget it.
